src/v2/api/dtos/CheckPermissionsDTO.ts

```diff
@@ -0,0 +1,17 @@
+import { IsArray, IsObject } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { PERMISSION } from '../../security/PERMISSION';
+
+export class CheckPermissionsDTO {
+    @ApiProperty({
+      enum: PERMISSION,
+      type: [PERMISSION],
+    })
+    @IsArray()
+      permissions: PERMISSION[];
+
+    @IsObject()
+    @ApiProperty()
```

Comment:
For example: if a permission will be a `admission.create`, there will be no need to use values field. Thus, this field must be optional in these cases.